omnisharptest/omnisharpIntegrationTests/testAssets/omnisharpTestAssets.ts

```diff
@@ -3,40 +3,17 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Shouldn't this file be named `OmnisharpTestAssetWorkspace` and the file above that currently has this name be maybe `ActiveTestAssetWorkspace`